quadrat/sass/blocks/_post-title.scss

```diff
@@ -0,0 +1,7 @@
+.wp-block-post-title {
+	margin-bottom: var(--wp--custom--gap--vertical);
```

Comment:
This looks like it belongs in Gutenberg. What do you think about opening an issue for it there?